device/mellanox/x86_64-mlnx_msn2700-r0/platform.json

```diff
@@ -0,0 +1,484 @@
+                "low_threshold": 10,
+                "low_critical_threshold": 0
+            },
+            {
+                "name": "thermal 38",
+                "high_threshold": 100,
+                "high_critical_threshold": 100,
+                "low_threshold": 10,
+                "low_critical_threshold": 0
+            },
+            {
+                "name": "thermal 39",
+                "high_threshold": 100,
+                "high_critical_threshold": 100,
+                "low_threshold": 10,
+                "low_critical_threshold": 0
+            },
+            {
+                "name": "thermal 40",
+                "high_threshold": 100,
+                "high_critical_threshold": 100,
+                "low_threshold": 10,
+                "low_critical_threshold": 0
+            }
+        ],
+        "sfps": [
+            {
+                "name": "sfp 1"
+            },
+            {
+                "name": "sfp 2"
+            },
+            {
+                "name": "sfp 3"
+            },
+            {
+                "name": "sfp 4"
+            },
+            {
+                "name": "sfp 5"
+            },
+            {
+                "name": "sfp 6"
+            },
+            {
+                "name": "sfp 7"
+            },
+            {
+                "name": "sfp 8"
+            },
+            {
+                "name": "sfp 9"
+            },
+            {
+                "name": "sfp 10"
+            },
+            {
+                "name": "sfp 11"
+            },
+            {
+                "name": "sfp 12"
+            },
+            {
+                "name": "sfp 13"
+            },
+            {
+                "name": "sfp 14"
+            },
+            {
+                "name": "sfp 15"
+            },
+            {
+                "name": "sfp 16"
+            },
+            {
+                "name": "sfp 17"
+            },
+            {
+                "name": "sfp 18"
+            },
+            {
+                "name": "sfp 19"
+            },
+            {
+                "name": "sfp 20"
+            },
+            {
+                "name": "sfp 21"
+            },
+            {
+                "name": "sfp 22"
+            },
+            {
+                "name": "sfp 23"
+            },
+            {
+                "name": "sfp 24"
+            },
+            {
+                "name": "sfp 25"
+            },
+            {
+                "name": "sfp 26"
+            },
+            {
+                "name": "sfp 27"
+            },
+            {
+                "name": "sfp 28"
+            },
+            {
+                "name": "sfp 29"
+            },
+            {
+                "name": "sfp 30"
+            },
+            {
+                "name": "sfp 31"
+            },
+            {
+                "name": "sfp 32"
+            }
+        ]
+    },
+    "interfaces": {
```

Comment:
should this be ports?

Reply:
`interfaces` is the expected key for port breakout infrastructure.